packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -293,6 +292,13 @@ async function createLanguageClient(extensionContext: vscode.ExtensionContext) {
     );
 
     if (languageClient) {
+      languageClient.onNotification('indexer/done', async () => {
+        await getTestOutlineProvider().refresh();
+        languageServerStatusBarItem.ready();
+
+        languageClientUtils.setStatus(ClientStatus.Ready, '');
+        languageClient?.errorHandler?.serviceHasStartedSuccessfully();
+      });
```

Comment:
can we pull this out into a utility method instead of having it in the activation method. testabilty.